demosplan/DemosPlanCoreBundle/Logic/Document/ElementsService.php

```diff
@@ -190,11 +190,13 @@ public function getHiddenElementsIdsForProcedureId(string $procedureId): array
             $this->getTopElements($procedureId, [], ['title' => $hiddenTitlesArray, 'deleted' => [false]]);
 
         // return IDs only:
-        return collect(array_merge($mapCategories, $hiddenByConfigCategories))->map(
-            fn ($element) =>
-                /* @var Elements $element */
+        return collect(array_merge($mapCategories, $hiddenByConfigCategories))
+            ->map(
+                fn ($element) =>
+                    /* @var Elements $element */
                 $element->getId()
```

Comment:
While pairing we determined the need to sort the element keys by their corresponding element titles, which was done [here](https://github.com/demos-europe/demosplan-core/pull/2637/files). Is that no longer needed? This PR now conflicts with the linked one. My understanding is that this ensured that hidden elements are sorted alphabetically at the very end of their group.